sources/it.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "url": "http://www.arpalazio.net/main/aria/sci/annoincorso/chimici.php",
+    "adapter": "arpalazio",
+    "name": "ARPALAZIO",
```

Comment:
They're using `ARPALAZIO`, an abbreviation of "Agenzia Regionale Protezione Ambientale Del Lazio".